src/jit/compile.h

```diff
@@ -11,22 +11,19 @@ struct MVMJitCode {
      * runtime (because we need a second dereference to figure the
      * labels out), but very simple for me now, and super-easy to
      * optimise at a later date */
-    MVMint32   num_labels;
-    void     **labels;
-
+    MVMint32       num_labels;
     MVMint32       num_bbs;
+    void         **labels;
     MVMint32      *bb_labels;
 
     MVMint32       num_deopts;
-    MVMJitDeopt    *deopts;
-
     MVMint32       num_inlines;
+    MVMJitDeopt    *deopts;
```

Comment:
Would be lovely if you added some comments to retain the information, which of the fields are related as you did in src/6model/reprs/MVMStaticFrame.h